### ISSUE TITLE
Display hook output

### ISSUE THREAD

**Issue by MikuZZZ** [jesseduffield/lazygit#863]
**Topic**
Is it possible to display actual console output of git hooks?

**Comment by lordlycastle** [comment]
Yes please. Also other command like push. Because currently you have to wait for it to fail then you see it. Hooks result is useful as you can see the tests running and it feels shorter because something is happening.

**Comment by shtlrs** [comment]
Is there any progress on this ?

I can see there's a command log window, but I can't find docs on how to open/close that, and checks its ouput easily via some keybinds or whatever.
